docs/hipify-clang.md

```diff
@@ -427,23 +427,23 @@ Run `Visual Studio 17 2022`, open the generated `LLVM.sln`, build all, build pro
 
 3. Ensure [`CUDA`](https://developer.nvidia.com/cuda-toolkit-archive) of minimum version 7.0 is installed.
 
-* Having multiple CUDA installations to choose a particular version the `DCUDA_TOOLKIT_ROOT_DIR` option should be specified:
+* Having multiple `CUDA` installations to choose a particular version the `DCUDA_TOOLKIT_ROOT_DIR` option should be specified:

```

Comment:
We could consider replacing 'should be' with 'must'.

Reply:
sure